common/src/main/java/uk/gov/gchq/palisade/service/request/DataRequestConfig.java

```diff
@@ -37,7 +34,7 @@
 public class DataRequestConfig extends Request {
     private User user = new User();
     private Justification justification = new Justification();
-    private Map<Resource, Rules> rules = new HashMap<>();
+    private MultiPolicy multiPolicy = new MultiPolicy();
```

Comment:
I liked the way we had it before as it kept all the Policy stuff in the Policy service and all the rest of the Palisade ecosystem cares about is the Rules.